litex/build/xilinx/vivado.py

```diff
@@ -203,6 +215,7 @@ def _constrain(self, platform):
 
     def build(self, platform, fragment, build_dir="build", build_name="top",
             toolchain_path=None, source=True, run=True, **kwargs):
+        synth_mode = kwargs.get('synth_mode', 'yosys')
```

Comment:
Shouldn't this be `kwargs.get('synth_mode', 'vivado')` for backwards compatibility?

Reply:
Correct, this will default to Vivado when it is out of WIP.